src/domain/relation/establish.ts

```diff
@@ -0,0 +1,15 @@
+
+import RelationData from './data/RelationData';
+import create from './data/create';
+import exists from './data/exists';
+import RelationAlreadyExists from './errors/RelationAlreadyExists';
+
+export default async function establish(followerId: string, followingId: string): Promise<RelationData>
+{
```

Comment:
```suggestion
import Requester from '../authentication/Requester';

export default async function establish(requester: Requester, followingId: string): Promise<RelationData>
{
    const followerId = requester.id;
    
```

We only want to allow creating relations for the requester (authenticated user). Otherwise relations can be created for others using the RPC API.